typings/number_format.d.ts

```diff
@@ -52,7 +52,7 @@ declare module "react-number-format" {
     type?: "text" | "tel";
     isAllowed?: (values: NumberFormatValues) => boolean;
     renderText?: (formattedValue: string) => React.ReactNode;
-    getInputRef?: (el: HTMLInputElement) => void;
+    getInputRef?: ((el: HTMLInputElement) => void) | React.Ref<any> | string;

```

Comment:
Passing getInputRef as string will break. Why do we want to keep that as option ?

Reply:
Oh sorry.
I think supporting string is not good, too.
I want to support React.Ref<any> only.